src/foam/nanos/om/DAOOMLoggerCopySink.java

```diff
@@ -0,0 +1,47 @@
+/**
```

Comment:
This file is added in https://github.com/foam-framework/foam2/pull/2079 as well, just a heads up. We don't want to add it twice.